package.json

```diff
@@ -64,10 +64,6 @@
     "src/**/*.scss": [
       "scssfmt",
       "git add"
-    ],
-    "src/**/*.go": [
-      "golangci-lint run -c .golangci.yml --fix",
-      "git add"
```

Comment:
`check:backend` only reports error. Since `--fix` option fixes actually our go codes in pre-commit by husky, so I think we need these lines.

Reply:
Did you try to modify Go files and push the commit? It fails for me :/

Reply:
Yes, of course. I confirm its behavior in several commits on local and on travis at previous PR. If your problem means this [ https://travis-ci.org/kubernetes/dashboard/jobs/499890953#L320 ], could you check my comment [ https://github.com/kubernetes/dashboard/pull/3617#discussion_r262768339 ] ?